### DISCUSSION TITLE
Mute / Block individual users

### DISCUSSION THREAD

**Discussion by hahmadia** [mattermost/community#148] [Practical Ideas]
I think a pretty generic function we are missing. We can mute a channel but what about a particular user who spamming but you don't wanna have to completely mute the channel. This is similar to Discorda blocking a user feature. With the mute infrastructure in place, I think this is something that we can get in without too much pain.

**Comment by esethna** [comment]
Makes sense to get a more granular level of mute that just the entire channel level

**Comment by chetanyakan** [comment]
Reporting a user who is spamming you.

**Reply by hahmadia** [reply]
That have to be it's own feature. Something like report them and the system admin of the server gets notified when x reports have been sent. 

This is more if the server admin is AFK or away and this feature fails to work then you can just mute them without needing a admin to do this.

**Reply by chetanyakan** [reply]
Yes, but reporting might help if the user is spammìng the entire channel.

We could have a dedicated moderators channel where all reports are sent.
